### PR TITLE
Spree to solidus renaming: first step

### PR DESCRIPTION
**Pull request by dfranciosi**
Some models still need to be updated on solidus gem, I'll work on it afterwards.

### CODE REVIEW

File: solidus_globalize.gemspec [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
-  s.version     = SpreeGlobalize.version
+  s.name        = 'solidus_globalize'
+  s.version     = SolidusGlobalize.version
   s.summary     = 'Provides globalization features for Spreecommerce.'
   s.description = s.summary
 
   s.author      = 'Sean Schofield'
   s.email       = 'sean.schofield@gmail.com'
-  s.homepage    = 'http://spreecommerce.com'
+  s.homepage    = 'http://soliduscommerce.com'
```
Comment (caesarsol) [inline solidus_globalize.gemspec]: :-1: !

Comment (dfranciosi) [inline solidus_globalize.gemspec]: Duoh! That's true, mate!

File: solidus_globalize.gemspec [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
-  s.name        = 'spree_globalize'
-  s.version     = SpreeGlobalize.version
+  s.name        = 'solidus_globalize'
+  s.version     = SolidusGlobalize.version
   s.summary     = 'Provides globalization features for Spreecommerce.'
   s.description = s.summary
 
   s.author      = 'Sean Schofield'
```
Comment (tvdeyen) [inline solidus_globalize.gemspec]: Nope!

File: solidus_globalize.gemspec [hunk 3]
```diff
@@ -25,7 +25,6 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'friendly_id-globalize'
   s.add_runtime_dependency 'globalize', '~> 5.0.1'
-  s.add_runtime_dependency 'spree_i18n', '~> 3.1.0.beta'
```
Comment (tvdeyen) [inline solidus_globalize.gemspec:28]: `'solidus_i18n', '~> 1.0'`

Comment (dfranciosi) [inline solidus_globalize.gemspec:28]: :ok_hand:

File: solidus_globalize.gemspec (hunk 2 above)
Comment (dfranciosi) [inline solidus_globalize.gemspec]: Why? Am I missing something? :unamused:

File: solidus_globalize.gemspec (hunk 3 above)
Comment (dfranciosi) [inline solidus_globalize.gemspec:28]: mmm...here, discussing with @alepore we're thinking if it's better use solidus_i18n from github master considering the edge status of both gems. What is your idea?

**Comment by tvdeyen** [comment]
It will not work unless you add the dependency inside the gemspec. The Gemfile does not get parsed by bundler outside of the gem itself. The Gemfile is only for your local gem development environment.

File: solidus_globalize.gemspec (hunk 2 above)
Comment (tvdeyen) [inline solidus_globalize.gemspec]: Sean is not Author of this gem. This is hence the old spree extension generator.

Comment (dfranciosi) [inline solidus_globalize.gemspec]: @tvdeyen ah sorry, I thought you was talking about my diffs, what about me, you and @alepore at the moment? Is it ok for you?

File: solidus_globalize.gemspec (hunk 3 above)
Comment (dfranciosi) [inline solidus_globalize.gemspec:28]: ok, clear enough :+1:

**Comment by tvdeyen** [comment]
No. I don't need to be mentioned as author. @alepore for sure. And just look into the git history, who ever made lots of contributions lately.

**Comment by alepore** [comment]
merging as starting point

**Pull request merged**